arm_robots/src/arm_robots/victor.py

```diff
@@ -76,6 +78,8 @@ def fill_using(joint_ordering: Sequence[str]):
 
 
 class BaseVictor(BaseRobot):
+    JOINT_LIM = np.array([math.pi * d / 180 for d in [170, 120, 170, 120, 170, 120, 175]])
```

Comment:
blehhhhh yet another place where we define joint limits :(

Reply:
Not sure where else we have joint limits

Reply:
the URDF